### PR TITLE
[DOM-21812] Fix project create method

### PR DESCRIPTION
**Pull request by abhijeet2096**
This P.R fixes the Project Create method.

### Link to Jira
https://dominodatalab.atlassian.net/browse/DOM-21812

### CODE REVIEW

File: domino/domino.py [hunk 1]
```diff
@@ -268,19 +268,17 @@ def deployment_version(self):
...
+            'projectName': project_name,
+            'ownerOverrideUsername': owner_username
         }
-        response = self.request_manager.post(url, data=request, allow_redirects=False)
-        disposition = parse_play_flash_cookie(response)
-        if disposition.get("error"):
-            raise Exception(disposition.get("message"))
-        else:
-            return disposition
+        headers = {
```
Comment (ddl-alexpanin) [inline domino/domino.py:277]: Can you explain why is this required here and not in other cases?

Comment (abhijeet2096) [reply domino/domino.py:277]: Hey Alex, I checked our server code, we are applying these CSRF filters on `POST` request with contentType `"application/x-www-form-urlencoded", "multipart/form-data", "text/plain"`. This headers needs to be added in two more places in `python-domino` (fork_project and collaborators_add). I am thinking to do that in a separate P.R
